firefox-ios/nimbus-features/homescreenFeature.yaml

```diff
@@ -35,6 +35,22 @@ features:
           },
           "pocket-sponsored-stories": false
         }
+  prefer-switch-to-open-tab:
+    description: >
+      Feature for managing the roll out of functionality that prefers switching to an already open tab instead of opening a duplicate tab.
+    variables:
+      enabled:
+        description: >
+          Enables the feature to automatically switch to an existing tab with the same content instead of opening a new one.
+        type: Boolean
+        default: false
+    defaults:
+      - channel: beta
+        value:
+          enabled: false
+      - channel: developer
+        value:
+          enabled: true
```

Comment:
I think there may have been some miscommunication here. This would be part of the homescreen feature configuration. So it would be a variable inside the `homescreenFeature` on line 2, like `sections-enabled` is.

Reply:
Third time's a charm. 😬